Tests/UIKitTests/UILayoutPriorityExtensionsTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UILayoutPriorityExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by diamantidis on 8/19/18.
+//  Copyright © 2018 SwifterSwift. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
```

Comment:
This `#if` should match the one used in the implementation file.

Reply:
I have updated the implementation file `#if`, since `UILayoutPriority` is only available on `iOS` and `tvOS`